File: platforms/android/Dezel/dezel/src/main/java/ca/logaritm/dezel/view/trait/TouchCancelable.kt
package ca.logaritm.dezel.view.trait

import android.view.MotionEvent

/**
 * @interface TouchCancelable
 * @since 0.7.0
 */
public interface TouchCancelable {

    /**
     * @method cancelTouchEvent
     * @since 0.7.0
     */
    fun cancelTouchEvent(event: MotionEvent)

}

File: platforms/ios/Dezel/Dezel/View/Trait/TouchCancelable.swift
/**
 * @protocol TouchCancelable
 * @since 0.7.0
 */
public protocol TouchCancelable {

    /**
     * @method cancelTouchEvent
     * @since 0.7.0
     */
    func cancelTouchEvent(touches: Set<UITouch>, with event: UIEvent)

}
